frontend/packages/dev-console/src/components/topology-list/TopologyToggleIcon.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import { chart_global_Fill_Color_900 as Color900 } from '@patternfly/react-tokens';
+import { Tooltip } from '@patternfly/react-core';
+
+import './TopologyToggleIcon.scss';
+
+interface TopologyToggleIconProps {
+  url: string;
+  icon: React.ReactNode;
+  tooltipText: string;
+}
+
+const TopologyToggleIcon: React.FC<TopologyToggleIconProps> = ({ url, icon, tooltipText }) => {
+  return (
+    <div className="odc-topology-toggle-icon">
+      <Tooltip position="left" content={tooltipText}>
+        <Link to={url} style={{ color: Color900.value }}>
```

Comment:
Is there not a better way to get a button look that is a link?
cc @jeff-phillips-18